tools/set-import/importer.ts

```diff
@@ -404,9 +404,9 @@ const SMOGON = {
 	bssseries2: 'battlestadiumsinglesseries2',
 } as unknown as { [id: string]: ID };
 
-const getAnalysis = retrying(async (u: string) => {
+const getAnalysis = retrying(async (o: FetchOptions) => {
 	try {
-		return smogon.Analyses.process(await request(u));
+		return smogon.Analyses.process(JSON.parse(await request(o)));
```

Comment:
```suggestion
		return smogon.Analyses.process(await request(o));
```
This was fixed https://github.com/pkmn/smogon/pull/15